Buttplug.Core/Messages/Messages.cs

```diff
@@ -82,10 +92,31 @@ public class DeviceMessageInfo
         [JsonProperty(Required = Required.Always)]
         public uint DeviceIndex;
 
+        [JsonProperty(Required = Required.Always, NullValueHandling = NullValueHandling.Ignore)]
+        public Dictionary<string, MessageAttributes> DeviceMessages =
+            new Dictionary<string, MessageAttributes>();
+
+        public DeviceMessageInfo(uint aIndex, string aName,
+            Dictionary<string, MessageAttributes> aMessages)
+        {
+            DeviceName = aName;
+            DeviceIndex = aIndex;
+            DeviceMessages = aMessages;
+        }
+    }
+
+    public class DeviceMessageInfo0

```

Comment:
Postfixing version numbers on classes seems kinda gross, but I don't have a better recommendation at the moment.

Reply:
It's not pretty, but it's clear